### PR TITLE
updated getting started tutorials

### PR DESCRIPTION
**Pull request by cdzialo**
-Modified all lessons within 'A_Getting_started' tutorial and corresonding images to adhere to new Human Standing template model and example.

### CODE REVIEW

**Review by melund — COMMENTED** [review]
A_Getting_started/_static/lesson2/image1.png seems very different from the original. Is that intended?

File: A_Getting_started/lesson1.rst [hunk 1]
```diff
@@ -1,11 +1,11 @@
-Lesson 1: Creating a Standing Model
+Lesson 1: Creating a Human Standing Model
 ==================================================
 
-The standing model can be found in the AMMR folder under
-Applications/Examples. The main file is called StandingModel.Main.any.
+The Human Standing model can be found in the AMMR folder under
```
Comment (melund) [inline A_Getting_started/lesson1.rst]: Yes. Just revert the "Human Standing" thing.

**Comment by melund** [comment]
Thanks a lot @cdzialo. This is exactly the update we want. Also nice that you fixed some the bad language :)

File: A_Getting_started/lesson1.rst (hunk 1 above)
Comment (cdzialo) [reply A_Getting_started/lesson1.rst]: Done. Just committed new files.

**Comment by melund** [comment]
Thanks a lot @cdzialo

**Pull request merged**